CRM/MembershipExtras/Test/Fabricator/MembershipType.php

```diff
@@ -2,8 +2,19 @@
 
 use CRM_Member_BAO_MembershipType as MembershipType;
 
+/**
+ * Class CRM_MembershipExtras_Test_Fabricator_MembershipType
+ */
 class CRM_MembershipExtras_Test_Fabricator_MembershipType {
```

Comment:
I agree with dome, don't see why not

Reply:
This fabricator was created quite some time ago by Tunbola, for the membership amount pro-rater class, which is why it has its own thing going. For starters, it uses the BAO directly, instead of the API to fabricate the membership type, and thus returns the BAO, instead of an array. Furthermore, it has this option to just build the BAO class, or to actually persist in the DB. I tried to make it more compliant to the base class, and have it extend it, but if we change it to our way of doing fabricators, we'd have to also change all the tests that depend on it. Maybe we can do this later?

Reply:
I think this is a good opportunity to change it before we start using it without thinking and then it become hard to change it @MiyaNoctem